daniel-v2/README.md

```diff
@@ -27,10 +28,18 @@
   - ✅✨ [Sitemap](https://wpdeveloperstaging.wordpress.com/docs/platform-features/sitemaps/)
   - ✅✨ [Jetpack Scan](https://wpdeveloperstaging.wordpress.com/docs/platform-features/jetpack-scan/)
   - ✅✨ [Account Security](https://wpdeveloperstaging.wordpress.com/docs/platform-features/account-security/)
+  - 🚧 [Automatic Data Center Failover](https://github.com/Automattic/dotcom-content/issues/218)
+  - 🚧 [Distributed Denial of Service (DDOS) and Web Application Firewall (WAF) protection](https://github.com/Automattic/dotcom-content/issues/219)
+  - 🚧 [Jetpack Search](https://github.com/Automattic/dotcom-content/issues/220)
+  - 🚧 [Akismet Anti-Spam](https://github.com/Automattic/dotcom-content/issues/221)
+  - 🚧 [Jetpack SEO Tools](https://github.com/Automattic/dotcom-content/issues/222)
```

Comment:
While this might be a good tool, I don't feel it's a deal breaker for selecting hosting. We can bundle it though under another sitemap item.

Reply:
OK, we can bundle it with the section of [Jetpack High-Level Walkthrough](https://github.com/Automattic/dotcom-content/issues/225)

Reply:
The Jetpack team said we shouldn't mention Jetpack SEO Tools at all at this point. We can remove it.

Reply:
Thanks @alexapeduzzi , I have removed it here [fafc52a](https://github.com/Automattic/wp-dev-docs/pull/8/commits/fafc52a44b7fe73c1cf3803ea31b41ff7ab8e02c)